src/components/PreferencesActions.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+import Vue from 'vue';
+export default Vue.extend({ name: 'preferences-actions' });
+</script>
+
+<template>
+  <div class="preferences-actions">
+    <button class="btn role-primary">
+      Apply
+    </button>
+    <button class="btn role-secondary">
+      Cancel
+    </button>
+  </div>
+</template>
+
+<style lang="scss" scoped>
```

Comment:
In a sense.. the dialog layout doesn't quite fit because it makes the assumption that padding should be applied around the entire layout. We can touch base before raising a PR for #2448 if necessary.